core/backend/src/BriefcaseManager.ts

```diff
@@ -432,6 +444,59 @@ export class BriefcaseManager {
     IModelJsFs.removeSync(changesetFile.pathname);
   }
 
+  /** @internal */
+  public static async revertTimelineChanges(db: IModelDb, arg: RevertChangesArgs): Promise<void> {
+    if (!db.isOpen || db[_nativeDb].isReadonly()) // don't use db.isReadonly - we reopen the file writable just for this operation but db.isReadonly is still true
```

Comment:
My point was that this comment is out of sync with code, as it is not true that 'we reopen the file writable just for `revertTimelineChanges` operation'. It seems like a copy - paste from pullAndApplyChangesets, where that comment is true as `pullChanges() in fact open file as writable just for pullAndApplyChangesets() operaion`. Suggest to change comment to something like : `// don't use db.isReadonly - it might be still true in cases when file is reopened writable just for this operation`

Reply:
I think I have fixed all you comments. See if you can close the comments.